src/app/utility/new-data-type-inline/new-data-type-inline.component.html

```diff
@@ -160,7 +135,37 @@
 					</div>
 				</td>
 			</tr>
+      <tr *ngIf="!showLabelAndDescription && model.domainType != 'PrimitiveType' && model.domainType != 'EnumerationType'">
+        <td colspan="2" style="text-align: right;">
+          <mat-slide-toggle (change)="toggleAdvanced()">Advanced Options</mat-slide-toggle>
+        </td>
+      </tr>
+      <tr *ngIf="showLabelAndDescription || model.domainType === 'PrimitiveType' || model.domainType === 'EnumerationType' || this.advanced">
+        <td class="detailsRowHeader">Data Type Label *</td>
+        <td>
+          <mat-form-field appearance="outline" class="full-width">
+            <input matInput name="label" [(ngModel)]="model.label" required>
+          </mat-form-field>
+        </td>
+      </tr>
+      <tr *ngIf="showLabelAndDescription || model.domainType === 'PrimitiveType' || model.domainType === 'EnumerationType' || this.advanced">
+        <td class="detailsRowHeader">Description</td>
```

Comment:
Why is the "Description" field considered an advanced option for Reference, Terminology, Code Set and Reference Data Model types but not others? Maybe there is a reason, but from me just looking at the form it confuses me.

Reply:
For the data types where we reference something else, the back-end can now supply a suitable data type name - e.g. "Reference to X".  That's what we tend to enter manually anyway.  And the description we'd leave blank - it's just a reference to something.  For primitives and enumerations, you're creating something new, and so you need to choose a name, at least, and ideally a description.